chroma-manager/Makefile

```diff
@@ -208,6 +208,13 @@ rpms: cleandist tarball
 	rm -rf _topdir
 	echo "jenkins_fold:end:Make Manager RPMS"
 
+install_requirements: requirements
+	echo "jenkins_fold:start:Install Python requirements"
+	pip install --upgrade pip;                              \
+	pip install --upgrade setuptools;                       \
+	pip install -Ur requirements.txt
+	echo "jenkins_fold:end:Install Python requirements"
+
 requirements: requirements.txt
```

Comment:
Why are we re-directing `requirements.txt` through `requirements` here?

It would be better to just remove this indirection and make all of the `requirements` dependencies `requirements.txt`